src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
lets add toString() method to Driver class (but do not add password field to it)